platform-stack/services

```diff
@@ -98,6 +98,120 @@ waitForIoTAgent () {
 	done
 }
 
+loraAgentSubscription() {
+	echo -e "\nSubscribing to mqtt\n"
+
+	curl --location --request POST 'localhost:4042/iot/services' \
+	--header 'fiware-service: openiot' \
+	--header 'fiware-servicePath: /' \
+	--header 'Content-Type: application/json' \
+	--data-raw '{
+	    "services": [
+	        {
+				"device_id": "urn:ngsi-ld:WasteContainer:martel-ttn-001",
+     			"entity_name": "urn:ngsi-ld:WasteContainer:martel-ttn-001",
+	            "entity_type": "WasteContainer",
+	            "apikey": "",
+	            "resource": "8CF957200005727C",
+	            "attributes": [
+	                {
+	                    "object_id": "temperature_1",
+	                    "name": "temperature",
+	                    "type": "Number"
+	                },
+	                {
+	                    "object_id": "relative_humidity_1",
+	                    "name": "relativeHumidity",
+	                    "type": "Number"
+	                }
+	                {
+	                    "object_id": "digital_in_1",
+	                    "name": "fillingLevel",
+	                    "type": "Number"
+	                }
+	                {
+	                    "object_id": "digital_in_2",
+	                    "name": "status",
```

Comment:
expression?

Reply:
Not working properly yet, there is something I'm missing